libs/platform/src/lib/platform.module.ts

```diff
@@ -23,6 +23,7 @@ import { PlatformInputModule, PlatformStepInputModule, PlatformInputGroupModule
 import { PlatformComboboxMobileModule } from './components/form/combobox/combobox-mobile/combobox-mobile.module';
 import { PlatformComboboxModule } from './components/form/combobox';
 import { PlatformThumbnailModule } from './components/thumbnail/public_api';
+import { PlatformObjectAttributeModule } from './components/object-attribute/public_api';
```

Comment:
Let's not import this from barrel files but use direct import pointing to the file